base/loading.jl

```diff
@@ -74,12 +74,21 @@ elseif Sys.isapple()
 else
     # Generic fallback that performs a slow directory listing.
     function isfile_casesensitive(path)
-        isfile(path) || return false
+        safe_isfile(path) || return false
         dir, filename = splitdir(path)
         any(readdir(dir) .== filename)
     end
 end
 
+function safe_isfile(file)
```

Comment:
Perhaps `isaccessiblefile`? "safe" doesn't mean anything to me, and notably, there actually is a real file there, but it cannot be examined

Reply:
Here "safe" refers to `isfile` operation (rather than file accessibility) i.e if there is an IOError then it returns  false instead of throwing it.

Reply:
I encountered this on relocated sysimg where the file didn't exist. 
And although,
```
julia> isfile("/root/doesn't/exist")
ERROR: IOError: stat("/root/doesn't/exist"): permission denied (EACCES)
```
` isfile("/root")` returns `false`, despite `cd("/root")` throwing stat-IOError
that's why I named it `safe_isfile` 

@vtjnash let me know if it should still be renamed to `isaccessible_file`, `isaccessible_path`, `isaccessible_dir`

Reply:
+1 for `isaccessible`.   "safe" implies that the ordinary version is "unsafe", but in Julia we ordinarily use "unsafe" for functions that might crash (not just throw).